tests/tensor/random/rewriting/test_basic.py

```diff
@@ -200,14 +199,18 @@ def test_inplace_rewrites(rv_op):
 def test_local_rv_size_lift(dist_op, dist_params, size):
     rng = shared(np.random.default_rng(1233532), borrow=False)
 
-    new_out, f_inputs, dist_st, f_rewritten = apply_local_rewrite_to_rv(
-        local_rv_size_lift,
-        lambda rv: rv,
-        dist_op,
-        dist_params,
-        size,
-        rng,
-    )
+    with pytest.warns(
+        FutureWarning,
+        match="test_value machinery is deprecated and will stop working in the future.",
+    ):
+        new_out, f_inputs, dist_st, f_rewritten = apply_local_rewrite_to_rv(
```

Comment:
The rewrite should not cause the warning, otherwise we know PyTensor users will be seeing the warning everywhere